pyClient/zeth/joinsplit.py

```diff
@@ -230,10 +230,10 @@ def gen_one_time_schnorr_vk_sk_pair() -> JoinsplitKeypair:
 
 def sign(
         keypair: JoinsplitKeypair,
-        hash_signature: str) -> int:
+        hash_to_be_sign: str) -> int:
```

Comment:
Either say `hash_to_sign` or `hash_to_be_signed`, but `hash_to_be_sign` is grammatically incorrect. I would stick to `hash_to_be_signed` to be consistent with the other variable name `data_to_be_signed`

Reply:
Fixed